CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added a new operator `XGBoost Training`. This operator allows to train a model in order to make predictions using the `XGBoost Prediction` operator.
```

Comment:
it's no longer an operator but a handler + task

Reply:
chore: update changelog entry to reflect current status
30bc265e